fdbclient/Knobs.cpp

```diff
@@ -148,6 +148,7 @@ ClientKnobs::ClientKnobs(bool randomize) {
 	init( HTTP_READ_SIZE,                     128*1024 );
 	init( HTTP_SEND_SIZE,                      32*1024 );
 	init( HTTP_VERBOSE_LEVEL,                        0 );
+	init( HTTP_REQUEST_ID_HEADER,                   "" );
```

Comment:
Is this something we're expecting ourselves or others to set regularly? If so, we've generally shied away from knobs for that. Or, at the very least, we provide another way to set the value that may then adjust the knob.

Reply:
After offline discussion it was decided we would leave this as a knob.